splunk_connect_for_snmp/traps.py

```diff
@@ -49,7 +49,7 @@
 SECURITY_ENGINE_ID_LIST = os.getenv("SNMP_V3_SECURITY_ENGINE_ID", "80003a8c04").split(
     ","
 )
-
+IPv6_ENABLED = True if "ipv6" in os.getenv("IPv6_ENABLED", "[IPv4]").lower() else False
```

Comment:
probably we can make it bool, anyway we assigning it on k8s based on ipFamily:

```
            - name: IPv6_ENABLED
               value: {{ .Values.traps.ipFamilies | quote }}
```

Reply:
changed to have it in chart